qgen/model.py

```diff
@@ -87,19 +89,16 @@
     if os.path.exists("model-{0}.index".format(i)):
         epoch = i
 
-writer = tf.summary.FileWriter(log_dir, session.graph)
-
-
 if epoch:
     saver.restore(session, "model-{0}".format(epoch))
 else:
     session.run(tf.global_variables_initializer())
 
 batch_index = 0
-for epoch in tqdm(range(epoch + 1, EPOCHS + 1), total=EPOCHS-epoch,
-                   desc="Training", unit=" epochs"):
-    summary = None
-    for batch in tqdm(training_data(), desc="Running batch", unit=" batches"):
+batch_count = None
+for epoch in trange(epoch + 1, EPOCHS + 1, desc="Epochs", unit="epoch"):
```

Comment:
It usually looks nicer with a space before the unit and with the unit pluralized: `unit=" epochs"`.  WDYT?
(same for the line below)

Reply:
I was seeing `6s/ epochs` which reads kind of weird.

Reply:
Then make your epochs faster!
lol yeah okay it does that when things are long. Okay LGTM then.